docs/README.md

```diff
@@ -173,50 +173,49 @@ $ ARCH=aarch64 scripts/mk.image
+|`LOGDNA_REDACT_REGEX`|`log.line_redact_regex`|List of regex patterns used to mask matching sensitive information (such as PII) before sending it in the log line.||
+|`LOGDNA_JOURNALD_PATHS`|`journald.paths[]`|List of paths (directories or files) of journald paths to monitor||
+|`LOGDNA_LOOKBACK`|`log.lookback`|The lookback strategy on startup|`none`|
+|`LOGDNA_K8S_STARTUP_LEASE`|`startup.option`|Determines whether or not to use K8 leases on startup|`never`|
+|`LOGDNA_USE_K8S_LOG_ENRICHMENT`|`log.use_k8s_enrichment`|Determines whether the agent should query the K8s API to enrich log lines from other pods.|`always`|
+|`LOGDNA_LOG_K8S_EVENTS`|`log.log_k8s_events`|Determines whether the agent should log Kubernetes resource events. This setting only affects tracking and logging Kubernetes resource changes via watches. When disabled, the agent may still query k8s metadata to enrich log lines from other pods depending on the value of `LOGDNA_USE_K8S_LOG_ENRICHMENT` setting value.|`never`|
+|`LOGDNA_LOG_REPORTER_METRICS`|n/a|Determines whether or not metrics usage statistics is enabled.|`never`|
+|`LOGDNA_DB_PATH`|`log.db_path`|The directory in which the agent will store its state database. Note that the agent must have write access to the directory and be a persistent volume.|`/var/lib/logdna`|
+|`LOGDNA_METRICS_PORT`|`log.metrics_port`|The port number to expose a Prometheus endpoint target with the [agent internal metrics](INTERNAL_METRICS.md).||
+|`LOGDNA_INGEST_TIMEOUT`|`http.timeout`|The timeout of the API calls to the ingest API in milliseconds|`10000`|
+|`LOGDNA_INGEST_BUFFER_SIZE`|`http.body_size`|The size, in bytes, of the ingest data buffer used to batch log data with.|`2097152`|
+|`LOGDNA_RETRY_DIR`|`http.retry_dir`|The directory used by the agent to store data temporarily while retrying calls to the ingestion API.|`/tmp/logdna`|
+|`LOGDNA_RETRY_DISK_LIMIT`|`http.retry_disk_limit`|The maximum amount of disk space the agent will use to store retry data. The value can be the total number of bytes or a human representation of space using suffixes "KB", "MB", "GB" or "TB", e.g. `10 MB` If left unset, the agent will not limit disk usage. If set to `0`, no retry data will be stored on disk.||
+|`LOGDNA_META_APP`|n/a|Overrides/omits `APP` field in log line metadata. [Examples](META.md)||
+|`LOGDNA_META_HOST`|n/a|Overrides/omits `HOST` field in log line metadata.||
+|`LOGDNA_META_ENV`|n/a|Overrides/omits `ENV` field in log line metadata.||
+|`LOGDNA_META_FILE`|n/a|Overrides/omits `FILE` field in log line metadata.||
+|`LOGDNA_META_K8S_FILE`|n/a|Overrides/omits `FILE` field in k8s log line metadata. Follows `LOGDNA_META_FILE`.||
+|`LOGDNA_META_JSON`|n/a|Overrides/omits `META` filed in log line metadata.||
+|`LOGDNA_META_ANNOTATIONS`|n/a|Overrides specific kay-value-pairs inside `ANNOTATIONS` field in log line metadata.||
+|`LOGDNA_META_LABELS`|n/a|Overrides specific kay-value-pairs inside `LABELS` field in log line metadata.||
```

Comment:
Nit: make `n/a` -> `N/A`

Reply:
Or, perhaps leave that field blank and make a note about it in the opening paragraph? Just so that the n/a's don't clutter up the table? Just a thought, totally up to you.

Reply:
I will make it blank and note in the paragraph.  Do you think I should make the other k8s ones blank?